src/implementations/twig/pages/page-home/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@ecl/twig-page-home",
+  "author": "European Commission",
+  "license": "EUPL-1.2",
+  "version": "4.6.3",
+  "description": "ECL Example page - Home",
+  "publishConfig": {
+    "access": "public"
```

Comment:
mmmh..these packages should be private, i think, someone might think that we are really providing a template for the home page and inner pages.